test/fixtures/invocation/async.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports.handler = async (event, context) => {
+  if (event && event.shoudFail) throw new Error('Failed on request');
```

Comment:
Should we use `shouldFail` property instead of `shoudFail`

Reply:
Great catch! Fixed